src/main/java/core/basesyntax/Report.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Report {
+    private final String date;
+    private final String name;
+    private final int hoursCount;
+    private final int salaryPerHour;
+
+    public Report(String report) {
+        String[] splitReportString = report.split(" ");
+        this.date = splitReportString[0];
+        this.name = splitReportString[1];
+        this.hoursCount = Integer.parseInt(splitReportString[2]);
+        this.salaryPerHour = Integer.parseInt(splitReportString[3]);
```

Comment:
Good idea to create entity of report, but this logic should not be in the constructor and no in this class